Rakefile

```diff
@@ -112,36 +116,34 @@ class VagrantTask < KitchenTask
 end
 
 namespace :test do
-  default_versions = %w[0.8.1.1 0.8.2.2 0.9.0.1 0.10.0.1]
+  default_versions = %w(0.8.1.1 0.8.2.2 0.9.0.1 0.10.0.1)
 
   def run_tests_for(versions, task_class)
-    $logger.info(sprintf('Running tests for versions: %s' ,versions.join(', ')))
-    failed_versions, done = [], false
-    until done do
+    $logger.info(format('Running tests for versions: %s', versions.join(', ')))
+    failed_versions = []
+    done = false
+    until done
       versions.each do |version|
-        $logger.info('Starting tests for v%s' % version)
+        $logger.info('Starting tests for v%s', version)
         task = task_class.new(version)
         if task.run.success?
-          $logger.info('Done testing v%s, run took %d seconds' % [version, task.duration])
+          $logger.info('Done testing v%s, run took %d seconds', version, task.duration)
         else
           $logger.info(task.output)
-          $logger.info('v%s failed, run took %d seconds, see output above ^' % [version, task.duration])
+          $logger.info('v%s failed, run took %d seconds, see output above ^', version, task.duration)
           if ENV.key?('yes')
             answer = ''
           else
-            print sprintf('%s :: Continue with the remaining versions? [Y/n]: ', Time.now.strftime('%Y-%m-%d %H:%M:%S'))
+            print format('%s :: Continue with the remaining versions? [Y/n]: ', Time.now.strftime('%Y-%m-%d %H:%M:%S'))
             answer = $stdin.gets.strip.downcase
           end
-          if answer.empty? || answer == 'y'
-            failed_versions << version
-            next
-          else
-            break
-          end
+          break unless answer.empty? || answer == 'y'
```

Comment:
> multiple conditions

I would agree is this was an if/elsif block, but its an if/else block so there is only one condition to chef, not multiple